javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/muzzle/matcher/ReferenceMatcher.java

```diff
@@ -143,50 +143,6 @@ private boolean doesMatch(ClassLoader loader) {
     }
   }
 
-  // for helper classes we make sure that all abstract methods from super classes and interfaces are
-  // implemented
-  private List<Mismatch> checkHelperClassMatch(Reference helperClass, TypePool typePool) {
```

Comment:
I think that the previous ordering made slightly more sense: methods were ordered by their usage. How about renaming the `checkMatch(Reference, TypeDescription)` method to `checkThirdPartyTypeMatch` (or something similar) without reordering them?

Reply:
👍 that's better